src/interpreter/index.ts

```diff
@@ -515,7 +514,23 @@ export class Interpreter implements Expr.Visitor<BrsType>, Stmt.Visitor<BrsType>
 
         if (satisfiedSignature) {
             try {
-                return callee.call(this, ...args);
+                let mPointer = this._environment.getM();
+
+                if (expression.callee instanceof Expr.DottedGet || expression.callee instanceof Expr.IndexedGet) {
```

Comment:
This is where the "magic" of the `m` pointer happens, for anyone reading this PR